go-controller/pkg/ovn/address_set/address_set.go

```diff
@@ -682,11 +674,11 @@ func splitIPsByFamily(ips []net.IP) (v4 []net.IP, v6 []net.IP) {
 	return
 }
 
-func ipsToString(ips []net.IP) []string {
-	// my kingdom for a ".values()" function
-	out := make([]string, 0, len(ips))
+// Takes a slice of IPs and returns a slice with unique IPs
+func ipsToStringUnique(ips []net.IP) []string {
```

Comment:
I agree this is much safer, curious if we hit the duplicate issue for address sets as well? where did we see this?